tools/pontoon/translate.md

```diff
@@ -0,0 +1,82 @@
+
+### Submit a Suggestion
+
+When a translator is in Suggest Mode, the blue 'Suggest' button will be located to the lower-right side of the editing space.
+
+![Blue button for suggestions](/assets/images/pontoon/translate/suggest_button.png)
+
+To suggest a translation, the translator should simply input the translation to the editing space and click `Suggest`. Once the translator has suggested the translation, the string's status will be updated from **missing** to **suggested**. This is indicated in the sidebar by the color of the checkbox next to the string.
+
+![Checkbox before suggestion](/assets/images/pontoon/translate/sidebar_presuggest.png) ![Checkbox after suggestion](/assets/images/pontoon/translate/sidebar_postsuggest.png)
+
+A notification will appear at the top of the window, indicating that your translation has been saved.
+
+![Translation saved notification](/assets/images/pontoon/translate/translation_saved_notif.png)
+
+*Note that a suggested translation exists only in the Pontoon database and not in the localized file or translation memories.*
+
+#### Manually Switch to Suggestion Mode
+
+To manually switch to Suggestion Mode, click the Settings gear to the lower-left side of the editing space and click the icon to the left of 'Make Suggestions' so that it becomes a green circle. The `Suggest` button should then appear to the lower-right side of the editing space.
+
+![Editing space settings](/assets/images/pontoon/translate/suggest_settings.png) ![Activate make suggestions](/assets/images/pontoon/translate/activate_suggest.png)
+![Make suggestions activated](/assets/images/pontoon/translate/activated_suggest.png)
+
+A notification will appear at the top of the window, indicating that `Make Suggestions` has been enabled.
+
+![Translation saved notification](/assets/images/pontoon/translate/translation_saved_notif.png)
+
+To switch to Review and Direct Translation Mode, go to the Settings menu by clicking on the gear and turn off the `Make Suggestions` option. The green `Save` button should appear to the lower-right side of the editing space.
+
+![The green save button](/assets/images/pontoon/translate/save_button.png)
+
+### Review a Suggestion
+
+To review a suggestion, search for suggested strings or click on a suggested string in the sidebar.
+
+If the suggested string is an acceptable translation, click on the `Save` button to the lower-right side of the editing space. This will approve the suggestion as a translation and save the string to the localized file.
+
+If the suggested string is **not** an acceptable translation, turn on `Make Suggestions` in settings. Once the `Suggest` button appears on the lower-right side of the editing space, edit the suggested string as appropriate and click `Suggest`. This will update the suggested string in the Pontoon database but will not save the string in the localized file. To see the old and new suggestions, click on the history tab below the editing space.
```

Comment:
This makes it sound like you can edit an existing suggestion, but you can't. You can only submit an alternative suggestion.

Also worth noting somewhere that the text area will show the most recent suggestion if there's no approved translation.

It's also missing how to interact with the history section, i.e. transforming an approved translation in a suggestion, approving another suggestion, deleting items